### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by ethdragon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,23 +6,6 @@
     <meta charset="utf-8">
     <link rel='dns-prefetch' href='//ajax.googleapis.com' />
     <link rel='dns-prefetch' href='//s.w.org' />
-    <link rel="alternate" type="application/rss+xml" title="Florida Institute for Cybersecurity Research &raquo; Feed" href="http://fics.institute.ufl.edu/feed/" />
```
Comment (ethdragon) [inline index.html]: these rss and calendar doesn't doing anything on your page, deleting

File: index.html [hunk 2]
```diff
@@ -6,23 +6,6 @@
...
-    <link rel="alternate" type="application/rss+xml" title="Florida Institute for Cybersecurity Research &raquo; Comments Feed" href="http://fics.institute.ufl.edu/comments/feed/" />
-    <link rel="alternate" type="text/calendar" title="Florida Institute for Cybersecurity Research &raquo; iCal Feed" href="http://fics.institute.ufl.edu/events/?ical=1" />
-    <style type="text/css">
```
Comment (ethdragon) [inline index.html]: this inline style doesn't do anything here, deleting

File: index.html [hunk 3]
```diff
@@ -114,7 +97,7 @@
             </nav>
             <!-- end #primary-nav -->
         </div>
-        <div class="jumbotron" style="padding: 2rem 1rem 0.25rem 1rem;">
```
Comment (ethdragon) [inline index.html]: this inline style has no visual effect

File: news.html [hunk 4]
```diff
@@ -127,10 +109,10 @@ <h2 id="news">News</h2>
                             <li>[Aug 2016] Our paper "Bear: A Framework for Understanding Application Sensitivity to OS (Mis)Behavior is accepted by ISSRE 2016".</li>
                             <li>[Mar 2016] I presented in <a href="http://fics.institute.ufl.edu/conference/past-programs/2016-program/"> FICS conference </a> with my work The Case for Unpredictability and Deception as OS Features. </li>
                             <li>[Jan 2016] I supervised teaching again on EEL 4930/EEL 5934 Cross-Layer Computer Security, University of Florida.</li>
-                            <li>[Oct 2015]<a href="http://www.alligator.org/news/campus/article_b28dc956-c31d-11e5-a1f9-03ae15b38446.html">Alligator</a> (Gainesville's major media) published our research on Chameleon.</li>
+                            <li>[Oct 2015] <a href="http://www.alligator.org/news/campus/article_b28dc956-c31d-11e5-a1f9-03ae15b38446.html">Alligator</a> (Gainesville's major media) published our research on Chameleon.</li>
```
Comment (ethdragon) [inline news.html]: Add space to be more consistent

File: index.html (hunk 1 above)
Comment (gracesrm) [reply index.html]: How to approve?

**Comment by gracesrm** [comment]
Fantastic work

**Pull request merged**